packages/scandipwa/src/component/Header/Header.component.js

```diff
@@ -418,13 +418,8 @@ export class Header extends NavigationAbstract {
             onSignIn
         } = this.props;
 
-        // This is here to prevent the popup-suspense from rendering
-        if (!showMyAccountLogin) {
-            return null;
-        }
-
         return (
-            <Suspense fallback={ this.renderAccountOverlayFallback() }>
```

Comment:
Please explain this modification.

Reply:
For this bug fix to work, MyAccountOverlay needed to also be included in the main page (otherwise MyAccountOverlay wasn't there when the page got refreshed so notifications couldn't be shown). However if it was included without any conditions, the popup started rendering, so it needed to have a similar modification like in the beginning but just for the fallback, so that MyAccountOverlay still is rendered in main page, but the popup isn't.

Reply:
What will happen if you leave the suspense value as it was before? I don't get the logic of rendering `null` as a suspense, that's why I am asking.

Reply:
If it stays the same the little window where my account is will render when main page is first opened or refreshed. Not a big issue, just doesn't look too pretty.